src/lazy/index.tsx

```diff
@@ -38,31 +45,37 @@ const createDeferred = (loader: Loader, sync: boolean) => {
 
 const lazyProxy = (
   loader: Loader,
-  { ssr = true, defer = PHASE.PAINT, id = () => '' }: Options = {}
+  {
+    ssr = true,
+    defer = PHASE.PAINT,
+    getCacheId = () => '',
+    moduleId = '',
+  }: Options = {}
 ) => {
-  const resolveId = id();
-  const resolveHash = hash(resolveId);
-  const deferred = createDeferred(loader, SETTINGS.IS_SERVER && ssr);
+  const isServer = isNodeEnvironment();
```

Comment:
Is there a reason why the usage of `isNodeEnvironment` vs `SETTINGS.IS_SERVER` is inconsistent when they should resolve to the same thing?

Reply:
SETTINGS.IS_SERVER is deprecated, I'll remove it